### ISSUE TITLE
Bookinfo broken in 1.5 branch

### ISSUE THREAD

**Issue by GregHanson** [istio/istio#21556]
**Bug description**
HTTP requests in the bookinfo for the istio.io task are broken.
```
$ curl -ivs http://${GATEWAY_URL}/productpage
*   Trying 169.60.241.30...
* TCP_NODELAY set
* Connection failed
* connect to 169.60.241.30 port 80 failed: Connection refused
* Failed to connect to 169.60.241.30 port 80: Connection refused
* Closing connection 0
```

Was afraid https://github.com/istio/istio/pull/21524 broke things when the tcp port was added to ingress gateway but I reverted that commit and rebuilt but problem persists

HTTP curls to bookinfo fail through ingress gateway:
https://preliminary.istio.io/docs/examples/bookinfo/#confirm-the-app-is-accessible-from-outside-the-cluster

TCP calls through ingress gateway work here so it appears to be effecting a subset of traffic:
https://preliminary.istio.io/docs/tasks/traffic-management/tcp-traffic-shifting/

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ X  ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ X ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

**Expected behavior**

**Steps to reproduce the bug**
checkout release-1.5 branch
currently need to `cherry-pick ee463d97588bbe119e52827530a0a950fba49ca5` to resolve `make docker` failures fixed by https://github.com/istio/istio/pull/20485

**Version (include the output of `istioctl version --remote` and `kubectl version` and `helm version` if you used Helm)**

**How was Istio installed?**

```
istioctl manifest apply --set profile=demo --set values.global.tag="test0226-2" --set values.global.hub=gihanson
```
**Environment where bug was observed (cloud vendor, OS, etc)**
IKS

**Comment by dgn** [comment]
Was this fixed by #21560? I'm working on extending the bookinfo test to cover this

**Comment by GregHanson** [comment]
@dgn it was resolved in release-1.5 branch yes.  What extension are you implementing?

**Comment by dgn** [comment]
@GregHanson I extended the bookinfo istio.io tests to actually test the reachability through the ingress-gateway - before, they didn't do that. next step will be to run them not just against the default profile, but all (or at least also demo)

**Comment by dgn** [comment]
This has been solved quite some time ago, before 1.5.0 release
